### PR TITLE
Fix mode-bar links navigation order with the tab key #4095 

### PR DESCRIPTION
**Pull request by Ifycode**
**This pull request makes the following changes:**
- Fixes ushahidi/platform#4095 

**Observation & steps taken:**
- Tab key navigation of the mode-bar links do not follow the order in which they are visually arranged on the page. 
- Made changes both in platform-client and platform-pattern library. Changed the arrangement of affected links in the html and styles. See the [associated platform-client pull request](https://github.com/ushahidi/platform-client/pull/1564).

**Testing checklist:**
- [x] Add assets folder from [platform-pattern-library containing the new changes](https://github.com/ushahidi/platform-pattern-library/pull/243/commits) made to the [platform-client which also contains some new changes](https://github.com/ushahidi/platform-client/pull/1564/commits) as instructed in documentation and run the project.
- [x] Log in to your account.
- [x] Navigate to links in the mode-bar using the tab key.
- [x] Check that outline moves from one link to the next in the order that the links are visually arranged (also test backwards with shift + tab).
- [x] Test after you log out from your account also. 

- [x] I certify that I ran my checklist

Fixes ushahidi/platform# .

Ping @ushahidi/platform

### CODE REVIEW

**Comment by Ifycode** [comment]
Related change to html can be found in this other pull request:
https://github.com/ushahidi/platform-client/pull/1564

**Comment by Angamanga** [comment]
@Ifycode Thank you! I will review shortly!

**Comment by Ifycode** [comment]
ok @Angamanga

File: pattern-library/partials/_mode-bar.html [hunk 1]
```diff
@@ -62,6 +62,51 @@
         </ul>
 
         <ul class="account-menu">
+            <li>
+                <a href="#" data-modal="collections">
+                    <svg class="iconic">
```
Comment (Angamanga) [inline pattern-library/partials/_mode-bar.html:67]: Could you try using tabindex here?

**Review by Angamanga — CHANGES_REQUESTED** [review]
Same thing as for the Platform-client, try switching to using tabindex instead of href if possible!

File: pattern-library/partials/_mode-bar.html (hunk 1 above)
Comment (Ifycode) [reply pattern-library/partials/_mode-bar.html:67]: Oh, ok. Here too. Will do so too Thanks.

**Comment by Ifycode** [comment]
Hi @Angamanga I dropped the problem with using tabIndex on the platform-client part of this pull request. Please check it out. Thanks. https://github.com/ushahidi/platform-client/pull/1564#issuecomment-713247651

**Comment by Ifycode** [comment]
Hi @Angamanga, I have changed the links to buttons. Please help review.

**Review by Angamanga — APPROVED** [review]
Thank you for your contribution!

**Comment by Angamanga** [comment]
@Angamanga To include in next PL release

**Comment by Ifycode** [comment]
Yay! OK. Thank you @Angamanga.

**Pull request merged**